### PR TITLE
Add spectra plot to prediction

### PR DESCRIPTION
**Pull request by jnwei**
Adding functionality to include spectra plots with predictions of spectra.

### CODE REVIEW

File: make_predictions.py [hunk 1]
```diff
@@ -58,7 +60,10 @@
     'Path to model checkpoint. If a directory, the most '
     'recent model checkpoint in this directory will be used. If a file, it '
     'should be of the form /.../name-of-the-file.ckpt-10000')
-
+tf.flags.DEFINE_bool(
+    'make_and_save_spectra_plots', True,
```
Comment (davidBelanger) [inline make_predictions.py]: I would just call this 'save_spectra_plots'

File: make_predictions.py [hunk 2]
```diff
@@ -130,20 +135,41 @@ def main(_):
...
+            # Dereference the singleton np string array to get the actual string.
+            key = key[0]
+            results[key] = prediction
+    else:
+        keys, predictions, true_spectra = fetched_values
+        for key, prediction, true_spectrum in zip(keys, predictions, true_spectra):
+            # Dereference the singleton np string array to get the actual string.
+            key = key[0]
+            results[key] = prediction
+            spectra_plot_file_name = plot_spectra_utils.name_plot_file(
```
Comment (davidBelanger) [inline make_predictions.py]: can you pull lines 159-168 into a helper function?

Comment (jnwei) [reply make_predictions.py]: Done.

File: make_predictions.py (hunk 1 above)
Comment (jnwei) [reply make_predictions.py]: Done.

**Comment by jnwei** [comment]
Finished incorporating suggestions.

**Pull request merged**